src/comments/comments.service.ts

```diff
@@ -41,11 +55,12 @@ export class CommentsService {
                 content
             }
         });
-        return comment;
+        const action = await this.actionsService.addAction('Create',userId,'Comment',comment.id,comment)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them